src/README.md

```diff
@@ -35,8 +35,11 @@ Note that if any of `vo`, `music`, or `sfx` are available features, `sound` will
 Also, all of these features are marked `false` by default.
 
 ## Handling State Change
+The SpringRoll Application class has a general [StateManager](./state) instance attached for managing important
+properties that can be changed via the container or outside environment. Property changes can be subscribed to allow the
```

Comment:
Developers can subscribe to property changes, allowing the game to react appropriately to the given situation.